tableaudocumentapi/connection.py

```diff
@@ -113,38 +108,48 @@ def username(self, value):
         self._username = value
         self._connectionXML.set('username', value)
 
-    ###########
-    # authentication
-    ###########
     @property
     def authentication(self):
         return self._authentication
 
-    ###########
-    # dbclass
-    ###########
     @property
     def dbclass(self):
+        """The type of connection (e.g. 'MySQL', 'Postgresql')."""
```

Comment:
The dbclass.py file lists my best guess at a comprehensive list that I extracted from our CPP code :), I can point them to that